### DISCUSSION TITLE
Welcome to akka.net Discussions!

### DISCUSSION THREAD

**Discussion by Aaronontheweb** [akkadotnet/akka.net#4660] [General]
## 👋 Welcome to Akka.NET!

  We’re using Github Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## Need Help?
* Need to report a bug with Akka.NET? [Open a new Github issue](https://github.com/akkadotnet/akka.net/issues/new)
* Need to understand how something works? [Read the Akka.NET documentation](https://getakka.net/), ask here, or [ask in the Akka.NET Gitter chat](https://gitter.im/akkadotnet/akka.net).
* Want to show off something you've built with Akka.NET? Do that here.
* Have some ideas for how to make Akka.NET better? Do that here.

**Comment by markvrc** [comment]
Hmm, I don't understand tho why it can't be all in one place. Why do we need to have 1) Gitter 2) issues 3) discussions if it can all be in issues? That's the problem almost all projects have. I understand that people want the best processes in a project. But what they forget is simplicity. Let's make things simple. Otherwise let's also open a slack chat for X purposes, telegram chat for Y purposes, our own messenger for Z, etc.

**Comment by coldwarrl** [comment]
@BadgerTheAdger I do not agree. Gitter - as every chat - is not persistent in that sense that it is very hard to find discussions about patterns, best practices, features and that stuff - Gitter is useful if you drop a question or so.

And I would see issues mainly for bug reporting or stuff, which involves the actual implementation project coding-wise.
Just my 2 cents here...

**Reply by Aaronontheweb** [reply]
Yeah, this is mostly why I wanted to add discussions - I love participating and answering questions in Gitter chat, but it can be hard to keep up with the volume of stuff in there and most importantly, I don't want the people who ask questions to get "missed" as chat goes by so quickly
